.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,6 +25,7 @@ body:
             - Extension:CMS-Professional
             - Extension:Custom-Products
             - Extension:DHL-App
+            - Extension:Digital-Sales-Room
```

Comment:
could you please add more affected areas? 
- Extension:Sales-Agent
- Extension:Recommendation
- Extension:AI-Image-Editor